src/bootstrap/src/core/build_steps/test.rs

```diff
@@ -1829,6 +1829,10 @@ NOTE: if you're sure you want to do this, please open an issue as to why. In the
             cmd.arg("--force-rerun");
         }
 
+        if builder.config.cmd.no_capture() {
+            cmd.arg("--nocapture");
```

Comment:
Remark: we should also rename this flag to `--no-capture` in compiletest, but I haven't gotten around to it.

Reply:
Shall we save that for a separate PR or want it done here?

Reply:
If you feel like it, it can just be in this PR since we're modifying the test infra pipeline here anyway.